README.md

```diff
@@ -972,162 +971,6 @@ function MyComponent() {
 
 - `called`
 
-## Testing and mocking
```

Comment:
why is this section gone?

Reply:
mea culpa - certainly wasn't intended - I may have missed updating locally after merging `master` into the feature branch. I checked the other later commits on this, and nothing else snuck into those.

Reply:
I've restored the missing content.